docs/trusted-resources.md

```diff
@@ -91,3 +91,36 @@ metadata:
 data:
   publickeys: "/etc/verification-secrets/cosign.pub, /etc/verification-secrets/cosign2.pub"
 ```
+
+#### Config key at VerificationPolicy
+VerificationPolicy supports SecretRef or encoded public key data.
+
+**Note:** Trusted Resources will first look up `VerificationPolicy` in the taskrun/pipelinerun's namespace, so the namespace should be the same of corresponding resources' namespace. If no `VerificationPolicy` it will check the system namespace, by default it is `tekton-pipelines`. If still no policy is found, it will try to fetch keys from configmap as described in last section.
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: VerificationPolicy
+metadata:
+  name: verification-policy
+  namespace: tekton-pipelines
+spec:
+  resourceAuthorities:
+      # pattern is used to filter out remote resources by their sources URL. e.g. if the resources from https://github.com/tektoncd/catalog.git, then the pattern should be set to https://github.com/tektoncd/catalog.git. ".*" will match any resources.
+    - pattern: ".*"
+      # authoritiyList defines a list of keys
+      authoritiyList:
+        - name: key1
+          key:
+            # secretRef refers to a secret in the cluster, this secret should contain public keys data
+            secretRef:
+              name:  tekton-pipelines
+              namespace: verification-secrets
+            # hashAlgorithm is the algorithm for the public key, by default is "sha256"
```

Comment:
what other values are supported?

Reply:
oh I will update this as well in the doc